

/controllers/positionController:
  ros__parameters:

    # PID controller for the angular velocity
    ## General value
    kp_angular: 1.0 
    ki_angular: 0.0
    kd_angular: 0.0


    ## Gain scheduling:  [start_zone, end_zone, kp_gain, ki_gain, kd_gain]
    gain_scheduling_angular: '[ [0,  10, 3.0, 4.2, 0.0], 
                                [10, 30, 1.0, 0.1, 0.0], 
                                [30, 60, 7.7, 0.0, 0.0], 
                                [60, 120, 4.8, 0.0, 0.0], 
                                [120, 150, 3.5, 0.0, 0.0]
                                [150, 180, 2.5, 0.0, 0.0]]'     
    
    # PID controller for the linear velocity
    kp_linear: 4.0        # Proporcional gain 
    ki_linear: 0.0        # Integrative gain
    kd_linear: 0.0        # Derivative gain

    # Velocity limits 
    max_linear_vel: 3.0   # Max linear velocity in a straight line 
    min_linear_vel: 0.0   # Min linear speed for rotational movement
    
    # Debug / test condition 
    use_sim_time: false   # Use time from simulation
    debug: true          # Enable prints 
    frequency: 6         # Node frequency 
    
    use_global_params: false      # Disable dependences from another nodes 
